powerlevel9k.zsh-theme

```diff
@@ -290,9 +290,15 @@ prompt_aws() {
 }
 
 # Segment to indicate background jobs with an icon.
+set_default POWERLEVEL9K_BACKGROUND_JOBS_VERBOSE true
 prompt_background_jobs() {
-  if [[ $(jobs -l | wc -l) -gt 0 ]]; then
-    "$1_prompt_segment" "$0" "$2" "$DEFAULT_COLOR" "cyan" "" 'BACKGROUND_JOBS_ICON'
+  local background_jobs_number=$(jobs -l | wc -l)
```

Comment:
Just found a little glitch: The default `wc` from OSX outputs additional whitespaces.. You could strip them like this:
`local background_jobs_number=${$(jobs -l | wc -l)// /}`